homeassistant/components/samsungtv/bridge.py

```diff
@@ -37,20 +41,24 @@
 )
 
 
-def mac_from_device_info(info):
+def mac_from_device_info(info: Any) -> str | None:
```

Comment:
`info` can probably be typed to be a `dict` of some kind

Reply:
Done